### ISSUE TITLE
Post-Processing Effects & Custom_Filtering Shaders

### ISSUE THREAD

**Issue by TRR95** [EKA2L1/EKA2L1#199]
Equal parts Critical and just something that would be nice to have. some games will benefit greatly from Post-Processing Effects and or Custom Filtering Shaders. It would make it easier to play games for long periods of time on larger screens, while others will just look nicer overall. 

I've tag this as one of the "Long Term Goals" of EKA2L1 since there is still work needs to be done on the graphics side of the emulator before we can add any additional effects or modifications to the games. 

 NOTE: Custom Filtering Shaders was originally requested by dmolina007#9747

**Comment by 1755917579** [comment]
You want to change the resolution

**Comment by 1755917579** [comment]
I want to change the game resolution

**Comment by pent0** [comment]
@1755917579 Hey, Symbian games hardcoded the game screen most of the time, so changing the game resolution is not possible with much of the library. The only thing we can do is to apply filter when scaling the screen. 

If you not used to Github, if you have suggestions, open another issue. Thank you!
